docs/install/docker-security.md

```diff
@@ -141,3 +141,36 @@ If you encounter any `File /usr/share/elasticsearch/config/elasticsearch.yml has
 {: .note }
 
 Finally, you can open Kibana at http://localhost:5601, sign in, and use the **Security** panel to perform other management tasks.
+
+## Using certificates with Docker
+
+To use your own certificates in your configuration, add all of the necessary certificates to the volumes section of the Docker Compose file:
+
+```yml
+volumes:
+- ./root-ca.pem:/full/path/to/certificate.pem
+- ./admin.pem:/full/path/to/certificate.pem
+- ./admin-key.pem:/full/path/to/certificate.pem
+#Add other certificates
+```
+
+Remember that the certificates you specify in your Docker Compose file must be the same as the certificates listed in `elasticsearch.yml`. At a minimum, you should replace the existing certificates with new ones you've created on your own.
+
+```yml
+opendistro_security.ssl.transport.pemcert_filepath: new-node-cert.pem
+opendistro_security.ssl.transport.pemkey_filepath: new-node-cert-key.pem
+opendistro_security.ssl.transport.pemtrustedcas_filepath: new-root-ca.pem
+opendistro_security.ssl.http.pemcert_filepath: new-node-cert.pem
+opendistro_security.ssl.http.pemkey_filepath: new-node-cert-key.pem
+opendistro_security.ssl.http.pemtrustedcas_filepath: new-root-ca.pem
+```
+
+If you want to use a custom `elasticsearch.yml` file, you can specify that in the volumes section as well.
```

Comment:
The moment you replace these certificates, you *must* include a custom elasticsearch.yml file, so let's make this a command.